### PR TITLE
Add ExaCA version 1.1 

### PR DESCRIPTION
**Pull request by streeve**

### CODE REVIEW

File: var/spack/repos/builtin/packages/exaca/package.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Exaca(CMakePackage):
 
     version("master", branch="master")
     version("1.0.0", sha256="48556233360a5e15e1fc20849e57dd60739c1991c7dfc7e6b2956af06688b96a")
+    version("1.1.0", sha256="10106fb1836964a19bc5bab3f374baa24188ba786c768e554442ab896b31ff24")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/exaca/package.py:22]: The convention is to list newer numbered versions first so please move to between lines 21 and 22.

**Review by tldahlgren — COMMENTED** [review]
Confirmed new sha256 and rest LGTM except the placement of the new version directive.

File: var/spack/repos/builtin/packages/exaca/package.py (hunk 1 above)
Comment (streeve) [reply var/spack/repos/builtin/packages/exaca/package.py:22]: Done - thanks

**Review by tldahlgren — APPROVED** [review]

**Comment by tldahlgren** [comment]
Looks like one CI stack is having unready container/pod issues.

@spackbot run pipeline

**Comment by tldahlgren** [comment]
@MattRolchigo  Did you want to comment on this PR before it is merged?

**Review by MattRolchigo — APPROVED** [review]

**Comment by MattRolchigo** [comment]
Looks good to me

**Pull request merged**